src/CMakeLists.txt

```diff
@@ -122,44 +122,37 @@ set(GISH_H
     config.h
 )
 
-if(WIN32)
-    list(APPEND GISH_H video/glext.h)
-endif(WIN32)
-
 if(APPLE)
     list(APPEND GISH_SRCS sdl/SDLMain.m)
     list(APPEND GISH_H sdl/SDLMain.h)
 endif(APPLE)
 
-if(WIN32)
-    add_executable(freegish WIN32 ${GISH_SRCS} ${GISH_H})
```

Comment:
I would leave this in, since otherwise if you build from powershell with cmake, you will not get a WIN32 application. Then you'll have some weird extra console window showing while playing.

Reply:
Reverted to using WIN32, couldn't find a good solution. Was hoping cmake would be more helpful here.